plan/typeinferer.go

```diff
@@ -363,7 +363,6 @@ func (v *typeInferrer) handleFuncCallExpr(x *ast.FuncCallExpr) {
 	case ast.ConnectionID, ast.InetAton:
 		tp = types.NewFieldType(mysql.TypeLonglong)
 		tp.Flag |= mysql.UnsignedFlag
-	// time related
```

Comment:
the master branch have out of date when I submit pr and this file have change a lot, so I merge some modify and rebase to master cause this problem.I think this is useless.